### DISCUSSION TITLE
NoMachine VS Omniverse Streaming Client

### DISCUSSION THREAD

**Discussion by 1g0rrr** [isaac-sim/IsaacAutomator#14] [Q&A]
There is a manual way to install Isaac Sim on AWS: https://docs.omniverse.nvidia.com/isaacsim/latest/installation/install_advanced_cloud_setup_aws.html
and there is a notice that "Only the Omniverse Streaming Client is supported for the best experience streaming to an AWS EC2 instance over the internet."

So is there any difference between NoMachine and Streaming Client in terms of convinience and long term support? 
Do I need to prefere one approach over another to stream from AWS machine?

**Comment by myurasov-nv** [comment]
Hi @1g0rrr ! The NoMachine is best for interactive sessions, while streaming is for occasionally checking what your simulation is doing. The statement you have mentioned means that on AWS AMIs that we provide it's the best option. However on Automator you have more. :)
